include/ck/tensor_operation/gpu/device/gemm_specialization.hpp

```diff
@@ -17,6 +18,15 @@ enum struct GemmSpecialization
     MKPadding,
     NKPadding,
     MNKPadding,
+    // Gemm + Gemm
```

Comment:
Better use 2 separate gemm_specialization for first and second GEMM

Reply:
There is 4 parameters M/N/K/O that are sufficient to describe Gemm+Gemm size. Gemm specialization merely follows that established convention. For some level of consistency, we're left with two options:
1. MNK size for Gemm0 and Gemm1, MNK padding for Gemm0 and Gemm1
2. MNKO size, MNKO padding (current)